src/dodal/devices/util/lookup_tables_apple2.py

```diff
@@ -347,51 +312,7 @@ def available_pol(self) -> list[Pol]:
     def available_pol(self, value: list[Pol]) -> None:
         self._available_pol = value
 
-    def _update_gap_lut(self) -> None:
-        if self.gap_path is None:
-            raise RuntimeError("Gap path is not provided!")
-        file_contents = self.config_client.get_file_contents(
-            self.gap_path, reset_cached_result=True
-        )
-        self.lookup_tables.gap = convert_csv_to_lookup(
-            file_contents, lut_config=self.lut_config
-        )
-        self.available_pol = list(self.lookup_tables.gap.root.keys())
-
-    def _update_phase_lut(self) -> None:
-        if self.phase_path is None:
-            raise RuntimeError("Phase path is not provided!")
-        file_contents = self.config_client.get_file_contents(
-            self.phase_path, reset_cached_result=True
-        )
-        self.lookup_tables.phase = convert_csv_to_lookup(
-            file_contents, lut_config=self.lut_config
-        )
-
-    def update_lookuptables(self):
-        """
-        Update lookup tables from files and validate their format.
-        """
-        LOGGER.info("Updating lookup table for gap.")
-        self._update_gap_lut()
-        if self.phase_path is None:
-            LOGGER.info("Generating lookup table for phase.")
-            self._generate_phase_lut()
-
-        else:
-            LOGGER.info("Updating lookup table for phase.")
-            self._update_phase_lut()
-
-    def _generate_phase_lut(self):
-        for key in self.lookup_tables.gap.root.keys():
-            if key is not None:
-                self.lookup_tables.phase.root[key] = generate_lookup_table_entry(
-                    min_energy=self.lookup_tables.gap.root[key].limit.minimum,
-                    max_energy=self.lookup_tables.gap.root[key].limit.maximum,
-                    poly1d_param=(PhASE_POLY1D_PARAMETERS[key]),
-                )
-
-    def get_motor_from_energy(self, energy: float, pol: Pol) -> tuple[float, float]:
+    def get_motor_from_energy(self, energy: float, pol: Pol) -> float:
```

Comment:
Could: I am not 100% what it should be but get_motor_form_energy seem wrong now.
```suggestion
    def get_value_from_lookuptable(self, energy: float, pol: Pol) -> float:
```